README.md

```diff
@@ -165,4 +165,12 @@ poetry install
 Finally, run the replicator:
 ```
 poetry run cdf_fabric_replicator config.yaml
+```
+
+# Docker
+
+If you are running the Docker Container on ARM based Macs. You will need to use platform emulation for the Docker run command. You can do this by running the following command:
```

Comment:
I would say ARM based architecture rather than Macs. You can have Raspberry Pi or other non-Mac based ARM hardware.

Reply:
good call